tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -0,0 +1,31 @@
+"""!/usr/bin/env python"""
```

Comment:
Sorry, I meant, the comment below on lines 3-4 should be docstringed, not the shebang (this should _not_ be commented or docstringed)